sdk/spring-sdk/src/test/java/kalix/springsdk/badwiring/valueentity/IllDefinedValueEntity.java

```diff
@@ -18,8 +18,10 @@
 
 import kalix.javasdk.valueentity.ValueEntity;
 import kalix.springsdk.annotations.Entity;
+import kalix.springsdk.annotations.EntityType;
 import org.springframework.stereotype.Component;
 
-@Entity(entityKey = "id", entityType = "test")
+@Entity("id")
```

Comment:
EntityKey?

Reply:
Strange that not all of them were refactored. Those in the integration tests should have been included when I refactored using IntelliJ. 

Others are not visible to IntelliJ, like those in the samples.

Anyway, I will review them all.

Reply:
I think I forgot to run the refactoring in IntelliJ. That's why all this classes were still using the old name.